lib/ui/platform_dispatcher.dart

```diff
@@ -1366,6 +1469,16 @@ class _PlatformConfiguration {
 
   /// The system-reported default font family.
   final String? systemFontFamily;
+
+  /// A unique identifier for the platform configuration that this
+  /// [_PlatformConfiguration] represents.
+  ///
+  /// This identifier can be used to to prevent race conditions, when the
+  /// Flutter application synchronously calls a platform function, and the
+  /// platform's configuration has changed on the platform thread but a new
+  /// [_PlatformConfiguration] has not been created for the Flutter application.
+  /// See the [_getScaledFontSize] function for an example.
+  final int? configurationGeneration;
```

Comment:
generation seems like a weird name, maybe token or iteration?

Reply:
renamed to `configurationId`